tests/conftest.py

```diff
@@ -42,8 +42,10 @@ def _sample_svs(tmpdir_factory):
 
 @pytest.fixture(scope="session")
 def _sample_jp2(tmpdir_factory):
-    """Sample pytest fixture for JP2 images. Download ndpi image for pytest."""
+    """Sample pytest fixture for JP2 images. Download jp2 image for pytest."""
```

Comment:
"""Sample pytest fixture for JP2 images. 
Download jp2 image for pytest.

"""